doc/newsfragments/2867_changed.fix_json_split_export.rst

```diff
@@ -0,0 +1 @@
+Fix an issue in JSON exporter which puts assertions from synthesized cases in the wrong data structure.
```

Comment:
Fix the issue that assertions in synthesized testcase (setup/teardown etc.) are not rendered on UI report.